backend/globaleaks/models/l10n.py

```diff
@@ -242,6 +244,8 @@ class NotificationL10NFactory(ConfigL10NFactory):
         'admin_anomaly_disk_low',
         'admin_test_static_mail_template',
         'admin_test_static_mail_title',
+        'x509_certificate_expiration_mail_template',
```

Comment:
here like everywhere lets prefer "HTTPS" in place of "X509".

the reasons are two:
- HTTPS is a more common name for the user.
- X509 is just a packaging format, multiple exists and it is not used just for HTTPS Certificates